config_example.sh

```diff
@@ -213,3 +213,6 @@
 # To enable FakeIPA and run dev-env on a fake platform
 # export NODES_PLATFORM="fake"
 # export FAKE_IPA_IMAGE=192.168.111.1:5000/localimages/fake-ipa
+
+# Whether to use ironic-standalone-operator to deploy Ironic.
+# export USE_IRSO="true"
```

Comment:
I think this config is supposed to have default values shown, but quickly checking others, we have already mixed it. Up to you which one you want to have here.